tools/syz-covermerger/syz_covermerger.go

```diff
@@ -144,25 +148,25 @@ func mergeResultsToCoverage(mergedCoverage map[string]*covermerger.MergeResult,
 		lines := maps.Keys(lineStat.HitCounts)
 		slices.Sort(lines)
 
-		var linesInstrumented, hitCounts []int64
-		var instrumented, covered int64
 		for _, line := range lines {
-			instrumented++
-			linesInstrumented = append(linesInstrumented, int64(line))
-			hitCount := lineStat.HitCounts[line]
-			hitCounts = append(hitCounts, int64(hitCount))
-			if hitCount > 0 {
-				covered++
+			res[allManagers][fileName].AddLineHitCount(line, lineStat.HitCounts[line])
+			managerHitCounts := map[string]int{}
+			for _, lineDetail := range lineStat.LineDetails[line] {
+				manager := lineDetail.Manager
+				managerHitCounts[manager] += lineDetail.HitCount
+			}
+			for manager, managerHitCount := range managerHitCounts {
```

Comment:
Are you using `res[manager]` anywhere now?

Reply:
coveragedb.ManagersCoverage goes to dashapi and is then used at pkg/coveragedb/spanner.go:89.
I write per-manager data into the db there.

Reply:
Is dashapi able to handle objects of that size? Or are they not that big?

Reply:
It is not clear now. I see the compression there and just hope it'll work.
In case of problems you'll see one more PR to remove that 32MB limit.